plugins/rich-editor/src/scripts/blots/EmbedLoadingBlot.js

```diff
@@ -0,0 +1,28 @@
+import { BlockEmbed } from "quill/blots/block";
+import { t } from "@core/utility";
+
+export default class EmbedLoadingBlot extends BlockEmbed {
+    static create() {
+        const node = super.create();
+        node.classList.add('embed');
+        node.setAttribute('role', 'alert');
+
+        node.innerHTML = "<div class=\"embedLoader\">\n" +
```

Comment:
This would be a lot cleaner with a template strings:

```js
node.innerHTML = `<div class='embedLoader'>
                    <div class='embedLoader-box'>
                        <div class='embedLoader-loader'>
                            <span class='sr-only'>
                                ${t('Loading...')}
                            </span>
                        </div>
                    </div>
                </div>`;
```